src/features/bravery/components/RandomItems.jsx

```diff
@@ -14,12 +14,20 @@ export function RandomItems() {
   const ability =
     state.selectedRandomChampion?.spells?.[state.selectedRandomAbilityIndex];
 
+
+  let abilityTooltipLabel = <TooltipCard header={`Start: ${ABILITY_ARRAY[state.selectedRandomAbilityIndex]}`} body="Error Loading Ability"/>
+  let abilityAvatarSrc = <WarningTwoIcon />;
+  if (ability) {
+    abilityTooltipLabel = <TooltipCard header={ability.name} body={ability.description} />;
+    abilityAvatarSrc = `${SPELL_IMAGE_BASE_URL}${ability?.image?.full}`;
+  }
+
   return (
     <Flex marginTop={8} gap={2} justifyContent="space-between">
       <Box>
-        <Tooltip label={<TooltipCard header={ability.name} body={ability.description} /> || ABILITY_ARRAY[state.selectedRandomAbilityIndex]}>
+        <Tooltip label={abilityTooltipLabel} >
           <Avatar
-            src={`${SPELL_IMAGE_BASE_URL}${ability?.image?.full}`}
+            src={abilityAvatarSrc}
```

Comment:
```suggestion
            src={ability?.name ? `${SPELL_IMAGE_BASE_URL}${ability?.image?.full}` : <WarningTwoIcon />}
```

Rather just do ternary here. Cleaner than random ifs